src/Raven.Server/Web/System/ResourceNameValidator.cs

```diff
@@ -69,6 +69,11 @@ public static bool IsValidResourceName(string name, string dataDirectory, out st
                 errorMessage = $"The name '{name}' is forbidden for use!";
                 return false;
             }
+            if (name.Contains(".") && name.Any(char.IsLetterOrDigit) == false)
```

Comment:
What about `/` or `\` ?

Reply:
`/` and `\` aren't valid character for database name and are checked by earlier validation check. 

Although `/` character is allowed for index name so I added that index name must not start with `.` if it contains `/` in  https://github.com/ravendb/ravendb/pull/8983/commits/d9c51cdf63e21211f202b21e3fb4357131a8fd2c